app.js

```diff
@@ -0,0 +1,112 @@
+    divDetails2.innerHTML = `<img src=${data.project1.deskimg} style="height:616px; margin-top:78px; width:872px; position: relative;"alt="project image"/>;
+    <div id="${Object.keys(data)[c]}dis" class="closexl" style="position:absolute; top:2%; right: 5%;"><i class="fa fa-close" style="float:right; padding:12px; color:#000; font-size:xx-large;" ></i></div>
+    <div style="display: flex;">
+    <h2 style="font-family: 'Inter', sans-serif; font-size: 32px; width: 589px; font-weight: 700; line-height:44px; color: #172b4d; padding:12px;">${data.project1.head}</h2>
+    <div style="display: flex; gap:10px; padding: 12px;">
+    <button type="button"  class="javabtn" style="padding: 10px; color: #fff; font-size: 15px; line-height:20px; font-weight:600; font-family: 'Inter', sans-serif; background-color: #36b37f; height: 48px; border-radius:5px; display:flex; justify-content:center; align-items:center; outline:none; border:none; ">See live<i class="fa fa-arrow-right" style="color: #fff; padding: 12px 12px 12px 12px; "></i></button>
+    <button type="button" class="javabtn" style="padding: 10px; color: #fff; font-size: 15px; line-height:20px; font-weight:600; font-family: 'Inter', sans-serif; background-color: #36b37f; height: 48px; border-radius:5px; display:flex; justify-content:center; align-items:center; outline:none; border:none; ">See Source<i class="fa fa-github" style="color: #fff; padding: 12px 12px 12px 12px; "></i></button>
+  </div>
+  </div>
+  <ul style="list-style:none; margin-bottom:12px; display:flex; gap:8px; flex-wrap:wrap;">
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">Codekit</li>
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">GitHub</li>
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">Javasript</li>
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">Bootstrap</li>
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">Terminal</li>
+    <li style="padding: 8px 12px; background: #ebf0ee; border-radius: 5px; color: #3a4a42; font-size: 12px; line-height:16px; font-weight:600; font-family: 'Inter', sans-serif;">Codepen</li>
+  </ul>
+  <p style="width:876px; height:313px;color: #344563; font-size: 16px; line-height:30px; font-weight:400; font-family: 'Inter', sans-serif; margin-bottom:16px;">${data.project1.para}</p>
+  `;
+    challdesk(`${Object.keys(data)[c]}dis`);
+  });
+}
```

Comment:
- [ ] So, now you will have an array of objects instead of an object of objects so you will need to redesign your application code according to that. You can see this [code](https://github.com/HAMMAS-SALEEM/Portfolio/blob/master/script/app.js) to get an idea of what you are going to do :+1: